src/client.js

```diff
@@ -1846,9 +1844,6 @@ function _sendEventHttpRequest(client, event) {
     return client._http.authedRequest(
         undefined, "PUT", path, undefined, event.getWireContent(),
     ).then((res) => {
```

Comment:
whats the point of this passthrough then handler now?

Reply:
Good point, dropped.